yt/frontends/amrex/tests/test_field_parsing.py

```diff
@@ -12,6 +12,7 @@
         pytest.param("X(C12H24)", [("C", 12), ("H", 24)], id="molecule_2"),
         pytest.param("X(H2O)", [("H", 2), ("O", 1)], id="molecule_3"),
         pytest.param("X(ash)", [("ash", 0)], id="descriptive_name"),
+        pytest.param("X(P_nse)", [("P_nse", 0)], id="descriptive_underscore"),
```

Comment:
I did a double take here: the underscore isn't descriptive by itself, or is it ?
```suggestion
        pytest.param("X(P_nse)", [("P_nse", 0)], id="with_an_underscore"),
```

Reply:
(the same applies to the other two test cases)